linux/ibus-keyman/src/engine.c

```diff
@@ -661,6 +680,12 @@ process_capslock_action(
 
     XSync(keyman->xdisplay, False);
   }
+#endif
+#ifdef GDK_WINDOWING_WAYLAND
+  // TODO
+  if (keyman->wldisplay) {
```

Comment:
Is this the TODO for caps lock modifiers?

Reply:
yes